turbo.json

```diff
@@ -1,39 +1,54 @@
 {
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["package.json"],
   "pipeline": {
     "build": {
-      "dependsOn": ["^build", "intl-compile", "codegen"],
+      "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "build:css": {
-        "outputs": ["src/assets/css/hydrogen.css"]
+      "inputs": ["src/**"],
+      "outputs": ["src/assets/css/hydrogen.css"]
     },
     "codegen": {
       "inputs": ["src/**/*.graphql", "../../packages/**/src/**/*.graphql"],
-      "outputs": ["src/api/generated.ts", "src/index.ts"]
+      "outputs": ["src/api/generated.ts", "src/index.ts"],
+      "outputMode": "new-only"
     },
     "dev": {
       "dependsOn": ["^build", "build:css", "intl-compile", "codegen"],
       "inputs": ["src/**", "package.json"],
-      "outputs": ["dist/**"]
+      "outputs": ["dist/**"],
+      "outputMode": "new-only"
     },
     "watch": {
       "dependsOn": ["build:css", "intl-compile", "codegen"],
-      "cache": false
+      "cache": false,
+      "persistent": true
     },
     "build:intl-cli": {
-      "outputs": ["dist/cli.js"]
+      "inputs": ["src/cli.ts"],
+      "outputs": ["dist/cli.js"],
+      "outputMode": "new-only"
     },
     "intl-extract": {
-      "outputs": ["src/lang/en.json"]
+      "inputs": ["src/**"],
+      "outputs": ["src/lang/en.json"],
+      "outputMode": "new-only"
     },
     "intl-compile": {
+      "inputs": ["src/**"],
```

Comment:
Yeah, this hurts that we can't use en.json here.

Reply:
I think it may be working in [update inputs to make it more specific](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5909/commits/f0fe86dd9c7b0dd8711cd161a2668717d6cdb8a2) 🤔

Reply:
You've added en.json but that's not a tracked file, right?

Reply:
So, when I was reading the documentation very carefully I noticed it says ["by default all non-git-ignored files in the workspace folder"](https://turbo.build/repo/docs/core-concepts/caching#:~:text=by%20default%20all%20non%2Dgit%2Dignored%20files%20in%20the%20workspace%20folder).

The way Interpreted this is that adding untracked files to the inputs property should override this and allow us to watch for changes in untracked files. From my testing, it **_seems_** to work